sdks/python/apache_beam/io/gcp/pubsub.py

```diff
@@ -61,29 +61,43 @@ class PubsubMessage(object):
     attributes: (dict) Key-value map of str to str, containing both user-defined
       and service generated attributes (such as id_label and
       timestamp_attribute). May be None.
+    message_id: (string) Message_Id as generated by PubSub
+    publish_time: (timestamp) Published time of message as generated by PubSub
+                  as per google.protobuf.timestamp_pb2.Timestamp
+
+    N.B. message_id and publish_time are not yet populated by Windmill service
```

Comment:
Users will probably be not familiar windmill service. Should we rephrase this as:

"TODO(BEAM-7819):  message_id and publish_time are not populated on Dataflow runner"

Reply:
Ah ok.  WIll update.